scanner/keppel/scanner/scanner.go

```diff
@@ -221,6 +236,36 @@ func (s *Scanner) GetTrivyReport(account string, repository string, manifest str
 
 }
 
+// extractImageInfo extracts image registry, image repository and the account name
+// from a container image
+func (s *Scanner) ExtractImageInfo(image string) (ImageInfo, error) {
+	// Split the string to remove the tag
+	parts := strings.Split(image, ":")
```

Comment:
The Tag is added?

Reply:
Maybe makes still sense to add it but the component name should be without tag

Reply:
In `ImageInfo` I can add `tag` as well.